src/cljs/org/broadinstitute/firecloud_ui/page/workspace/analysis/igv.cljs

```diff
@@ -0,0 +1,36 @@
+(ns org.broadinstitute.firecloud-ui.page.workspace.analysis.igv
+  (:require
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common :as common]
+    [org.broadinstitute.firecloud-ui.utils :as utils]
+    ))
+
+
+(defn- options [tracks]
+  (clj->js
+    {:genome "hg19"
+     :trackDefaults {:palette ["#00A0B0" "#6A4A3C" "#CC333F" "#EB6841"]
+                     :bam {:coverageThreshold 0.2
```

Comment:
1) do we send :bam info even on beds and vcfs?

2) might want to override defaults for track height - when viewing multiple tracks, it's awfully big.